website/content/specification/syntax/constraints.md

```diff
@@ -179,6 +179,21 @@ A constraint may have an OPTIONAL [`@level`](#level) attribute and/or an OPTIONA
 
 If defined, the `<message>` value MUST be a [Metaschema string value](/specification/datatypes#string). It MAY contain a Metapath expression templates that starts with `{`, contains a Metapath expression, and ends with `}`.  When evaluating a template Metapath expression, the context of the Metapath [evaluation focus](#constraint-processing) MUST be the failing value node.
 
+## `is-unique` constraints
+
+The `is-unique` constraint is a type of Metaschema constraint that restricts field or flag value(s) based on a component not being referenced more than once.
+
+The `id` attribute of an `is-unique` constraint specifies an identifier for the constraint.
+
+The `@target` attribute of an `<is-unique>` constraint specifies the node in a component definition whose value is distinct. The `is-unique` constraint MUST define a [`@target`](#target) with a Metapath expression.
+
+The `key-field` child element of an `<is-unique>` constraint defines the attribute or element value that is the key for each entry in the index. The `<key-field>` element MUST define a [`@target`](#target) attribute with a Metapath expression evaluated relative to [the evaluation focus](#constraint-processing) of each entry of the matches of the constraint's `@target`.
```

Comment:
We should avoid using XML terminology here. Instead, we should use Metaschema terms like flag, field, and assembly.

Reply:
Should this read,  "The `key-field` field  of an `<is-unique>'